test/reuse/util/browser/forward.spec.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const browser = require("../../../../lib/reuse/modules/util/browser");
+const { handleCookiesConsent } = require("../../../helper/utils");
+
+describe("Expect browser url to be equal with 'startUrl' after browser forward", function () {
+  it("Preparation", async function () {
+    await common.navigation.navigateToUrl("https://sapui5.hana.ondemand.com/1.99.0/");
+    await handleCookiesConsent();
+    const selector = {
+      elementProperties: {
+        viewName: "sap.ui.documentation.sdk.view.App",
+        metadata: "sap.m.IconTabFilter",
+        text: [
+          {
+            path: "i18n>APP_TABHEADER_ITEM_DOCUMENTATION"
+          }
+        ]
+      }
+    };
+    await ui5.userInteraction.click(selector);
+  });
+
+  it("Execution & Verification", async function () {
+    await util.browser.forward();
```

Comment:
not sure how this test works since "forward" requires a "back" before?

Better do the following:
- execute "back"
- check the new url
- execute "forward"
- check the url before

Reply:
I will implement the above steps

Reply:
done